### PR TITLE
[Checkmarx] Resolve "Client Use Of JQuery Outdated Version" warnings

### PR DESCRIPTION
**Pull request by alexhoffman617**
This PR removes the old hard coded versions of jquery and replaces all references to them with a reference to up to date jquery in node modules

---

## Additions

- Gulp script to expose jquery to application so HTML can import

## Removals

- Hard coded minified versions of jquery 1.9.1

## Changes

- point to node modules version of jquery

## How to test this PR

1.

## Notes and todos

-

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

#### Browser testing

Visually tested in the following supported browsers:
- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [ ] Safari on iOS
- [ ] Chrome on Android

#### Accessibility

- [ ] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [ ] Screen reader friendly
- [ ] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Does not introduce new lint warnings
- [ ] Flexible from small to large screens

### CODE REVIEW

**Review by anselmbradford — CHANGES_REQUESTED** [review]
Hey thanks for your work on this! I gotta say though, I'm really not keen to add jquery back into the base package.json and to add a dedicated copy task for it. I believe this is only used to support the "chosen" plugin that powers the drop-down [on credit card agreements](https://www.consumerfinance.gov/credit-cards/agreements/) and for the email validation and page scrolls when you click the buttons on [the supervision page](https://www.consumerfinance.gov/jobs/supervision/), both of which are legacy pages that should be migrated to a more modern approach.

What if we just updated the hardcoded version at cfgov/agreements/static/agreements/js/jquery-1.9.1.min.js and cfgov/legacy/static/nemo/_/js/jquery-1.9.1.min.js to satisfy checkmarx and left the legacy structure in place so it can be excised without touching the main build architecture?

For reference, there are a handful of issues in our internal github about migrating the five deprecated pages that use these legacy templates:

https://[GHE]/CFGOV/platform/issues/3700
https://[GHE]/CFGOV/platform/issues/3698
https://[GHE]/CFGOV/platform/issues/3699
https://[GHE]/CFGOV/platform/issues/3697
https://[GHE]/CFGOV/platform/issues/3695

**Comment by anselmbradford** [comment]
Removed jquery in one page here https://github.com/cfpb/consumerfinance.gov/pull/6030

**Comment by anselmbradford** [comment]
Removed jquery in base_update template here https://github.com/cfpb/consumerfinance.gov/pull/6031

**Review by Scotchester — CHANGES_REQUESTED** [review]
OK, we have reduced this PR to the one thing remaining that _needs_ jQuery (without a bigger refactoring effort), the credit card agreement database.

On reflection, I agree with Ans that it'd be better to just grab the latest minified jQuery and drop it in the same `cfgov/agreements/static/agreements/js/` location that we deleted 1.9.1 from, rather than add another wrinkle to our already-complex build processes. Since the Node dependency would be pinned, it's not saving us any work if a new vulnerability comes along; we'd still need to update the codebase to correct it.

File: yarn.lock [hunk 1]
```diff
@@ -11822,7 +11822,7 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-ua-parser-js@0.7.22, ua-parser-js@^0.7.18, ua-parser-js@^0.7.22:
+ua-parser-js@^0.7.18, ua-parser-js@^0.7.22:
```
Comment (Scotchester) [inline yarn.lock:11825]: Let's revert this change that no longer applies to anything else in the PR.

```suggestion
ua-parser-js@0.7.22, ua-parser-js@^0.7.18, ua-parser-js@^0.7.22:
```

**Review by Scotchester — APPROVED** [review]
Approved, pending one last tiny little fix!

**Pull request merged**